benchmarks/synchrobench/src/main/java/com/yahoo/oak/synchrobench/contention/benchmark/OpCounter.java

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak.synchrobench.contention.benchmark;
```

Comment:
This is a general comment:

This is very significant rewrite of the benchmarks structure for Oak project. I find it hard to get into every detail of such a big code update.

1. First concern: For next conference paper, will we be able to say that we evaluate our "something" using a "well known" synchrobench platform?  For example, even Nova for PLDI still runs evaluation using Oak's synchrobench. Nova runs on Ramy's fork so will not be immediately affected by this change. But still what do you think about divergence from synchrobench? Not that it was too similar from the beginning...